ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/css/datepickerwidget.css

```diff
@@ -154,7 +154,7 @@
 
 .cmp-adaptiveform-datepicker__calendar-icon {
     position: absolute;
-    top: 50%;
+    top: 41px;
```

Comment:
This will again cause issues with other view ports ? Have you verified other view ports manually

Reply:
Checked on different view ports and we have to give top in px only, if given in % icon moves to downside when error appears on the UI because it moves relative to the parent.